### PR TITLE
Add lombok

### PR DESCRIPTION
**Pull request by VNTee**
Add config for lombok and lombok, but you still need to install plugin for lombok in yours IDE.

How to install plugin in the IDEA:
1. File -> Settings -> Plugins -> Lombok plugin -> Install
2. Restart IDEA

May be you will need to enable annotation processor:
1. File -> Settings -> Build, Execution, Deployment -> Compiler -> Annotation Processors
2. Enable annotation processing

### CODE REVIEW

**Comment by imbirevkotlya** [comment]
please, provide some info in description, to explain how to enable annotation processor and lombok plugin in Intellij

**Comment by imbirevkotlya** [comment]
Also, provide please some example of using Lombok in code, for example create class Test and try to use lombok/delombok features

File: build.gradle [hunk 1]
```diff
@@ -23,6 +23,8 @@ repositories {
 }
 
 dependencies {
+    compileOnly "org.projectlombok:lombok:1.18.4"
+    testCompileOnly "org.projectlombok:lombok:1.18.4"
```
Comment (imbirevkotlya) [inline build.gradle]: Is it possible to use only 'provided' here?

Comment (VNTee) [reply build.gradle]: > Also, provide please some example of using Lombok in code, for example create class Test and try to use lombok/delombok features

Do we need to have useless extra classes in our project? I checked it out, it worked properly. I ll show lombok's features for my team today at the presentation.

Comment (VNTee) [reply build.gradle]: > 
> 
> Is it possible to use only 'provided' here?

Sure, it is possible, but lombok use annotations only at compile time, we dont wanna have it in our final product.

Comment (imbirevkotlya) [reply build.gradle]: okay, i suppose your presentation will be enough.
Also, agree about compile time

**Pull request merged**